### ISSUE TITLE
Benchmark Group members should use the same unit

### ISSUE THREAD

**Issue by djugei** [bheisler/criterion.rs#367]
currently benchmarks within the same benchmark group may use different time units for displaying, this makes it a bit harder to compare at a glance. for example one may use nano-seconds and one micro-seconds.

they should instead use one unit for the whole group.

**Comment by bheisler** [comment]
Hey, thanks for the suggestion and thanks for trying Criterion.rs!

I agree this would be nice to have. Unfortunately, I haven't had a lot of time to work on Criterion.rs lately. Pull requests would be welcome though.

**Comment by Mart-Bogdan** [comment]
That's indeed much needed!

I've just bumped into this

![image](https://user-images.githubusercontent.com/775038/235545548-b4b68a6e-d155-4e85-ba19-62516a4b1223.png)


I was looking at results and thinking that this is x2 times faster, but only after few hours I understood that it is actually 500x times slower

I just thought that **rustc** did some crazy optimization.

**Comment by Mart-Bogdan** [comment]
Perhaps we can force measurement units for whole benchmark in config/env/cli option? Not for groups but for whole process?
